### PR TITLE
Theming

### PR DESCRIPTION
**Pull request by nikhilmandlik**
TODO:

- [x] not able to change theme on (dev) Project depend on openmct using **openmct.install(**
eg: `openmct.install(openmct.plugins.Espresso());`
- [x] not able to change theme on (prod) Project depend on openmct using **openmct.install(**
eg: `openmct.install(openmct.plugins.Espresso());`

### CODE REVIEW

File: src/MCT.js [hunk 1]
```diff
@@ -325,6 +325,24 @@ define([
         });
     };
 
+    /**
+     * Get path to where assets are hosted.
+     * @memberof module:openmct.MCT#
+     * @method getAssetPath
+     */
+    MCT.prototype.getAssetPath = function () {
+        var constants = this.legacyBundle.extensions.constants;
```
Comment (akhenry) [inline src/MCT.js]: Let's not use Angular for this, we don't need it. Let's just store it on a local variable.

eg.
``` JavaScript
MCT.prototype.setAssetPath = function (assetPath) {
     this._assetPath = assetPath;
}
MCT.prototype.getAssetPath = function () {
     return this._assetPath || '/';
}
```

File: src/ui/layout/browse-bar.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
+.l-browse-bar {
```
Comment (akhenry) [inline src/ui/layout/browse-bar.scss]: Please remove this file altogether. These styles have been moved to a different file in  https://github.com/nasa/openmct/pull/2568#discussion_r356324689

File: src/plugins/themes/installTheme.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const dataAttribute = 'theme';
+
+export const installTheme = (openmct, themeName) => {
+    const currentTheme = document.querySelector(`link[data-${dataAttribute}]`);
```
Comment (akhenry) [inline src/plugins/themes/installTheme.js]: Extra credit!

File: src/plugins/themes/espresso.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { installTheme } from './installTheme';
+
+export default function plugin() {
+    return function install(openmct) {
+        installTheme(openmct, 'espresso');
```
Comment (akhenry) [inline src/plugins/themes/espresso.js]: Beautiful!

File: src/MCT.js (hunk 1 above)
Comment (nikhilmandlik) [reply src/MCT.js]: Ahh this is Angular stuff you were talking about. Let me change it.

File: src/MCT.js [hunk 5]
```diff
@@ -318,11 +316,17 @@ define([
      * @memberof module:openmct.MCT#
      * @method setAssetPath
      */
-    MCT.prototype.setAssetPath = function (path) {
-        this.legacyExtension('constants', {
-            key: "ASSETS_PATH",
-            value: path
-        });
+    MCT.prototype.setAssetPath = function (assetPath) {
+        this._assetPath = assetPath;
```
Comment (akhenry) [inline src/MCT.js]: Could we please conditionally add a trailing `/` if it's absent? This threw me when I tried to update the tutorial.

**Comment by akhenry** [comment]
Fixes #2507

**Comment by akhenry** [comment]
Looks great. Have filed a followup to add unit tests - https://github.com/nasa/openmct/issues/2576

## Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? N/A
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards?

**Pull request merged**